tox.ini

```diff
@@ -565,6 +571,7 @@ setenv =
 
 
 [flake8]
-ignore=W391,E231,E201,E202,E203,E261,E302,E128,E126,E124
+ignore=W391,E231,E201,E202,E203,E261,E302,E128,E126,E124,W503
```

Comment:
This was needed to prevent flake8 from complaining about https://github.com/DataDog/dd-trace-py/blob/c1805c2ffaa71770f20ee18690f6aa61ea801739/ddtrace/contrib/vertica/patch.py#L171-L174

See https://github.com/PyCQA/pycodestyle/issues/513 for more info.

Reply:
at some point I would like to remove 95% of these `ignore`s

Reply:
`s/95/100/` ;P